starpy/manager.py

```diff
@@ -118,7 +119,10 @@ def registerEvent(self, event, function):
         """
         log.debug('Registering function %s to handle events of type %r',
                   function, event)
-        if isinstance(event, (str, unicode, type(None))):
+        types = [str, type(None)]
+        if sys.version_info <= (3, 0):
+            types.append(unicode)
```

Comment:
Thats was my first thinking. I wanted to do it without add other dependence to code, this was the end idea.